pkgs/test/src/cli.ts

```diff
@@ -11,9 +13,46 @@ cli.command("launch")
     .description("Launch the Redo app")
     .action(async () => {
         console.log("Launching the app...")
+        const { outdated } = await isNewVersionAvailable(version)
+        if (outdated) { console.log("New Update is Avaiable. You can Upgrade to the latest version with redo update.") }
```

Comment:
re-do test will still always look for the app release matching its version. So we need to either mention that they will need to update the version in their package.json file or do it automatically (mentioning that it needs to be done is probably simpler for now)